src/js/components/FragmentList/FragmentListItem/fragmentListItem.style.js

```diff
@@ -15,93 +15,88 @@
  */
 
 import styled from 'styled-components';
+import {
+  COLOR_SUCCESS,
+  COLOR_ERROR,
+  COLOR_OTHER,
+  COLOR_UNPROCESSED,
+  COLOR_MISSING,
+} from '../../../helpers/graph/drawingHelper';
 
 export const FragmentListItemContainer = styled.div`
     display: flex;
-    border-bottom: 1px solid ${({ theme }) => theme.borderColor};
-    color: ${({ theme }) => theme.textColor};
-    background-color: ${({ active, theme }) => (active ? theme.fragmentActive : '')};
+    color: ${({ theme }) => theme.TEXT};
+    background-color: transparent;
     margin: 1px;
-    min-height: 40px;
+    height: 30px;
     align-items: center;
-    font-size: 14px;
+    font-size: 12px;
 
-    &:nth-child(2n + 1) {
-        background-color: ${({ theme }) => theme.oddFragmentBgColor};
-        background-color: ${({ expanded, theme }) => (expanded ? theme.fragmentHighlight : '')};
+    & > * {
+       background-color: ${({ active, theme }) => (active ? theme.TABLE_CELL_ACTIVE_BG : theme.TABLE_CELL_BG)};
     }
 
-    &:hover, &:focus{
-        background-color: ${({ theme }) => theme.fragmentHighlight};
+    &:hover, &:focus {
+      & > * {
+        background-color: ${({ theme }) => theme.TABLE_CELL_HOVER_BG};
+      }
     }
 `;
 
 export const StatusWrapper = styled.div`
-    height: 25px;
-    width: 25px;
-    margin: 2px;
+    height: 30px;
+    width: 30px;
+    margin: 1px;
```

Comment:
Why is it set to 1px? It looks a little bit fishy to me 🤔

Reply:
consistent with design: https://xd.adobe.com/view/0caae196-e833-4ace-57ad-e035865e8e69-7a9b/screen/e6cec8e4-62d6-48b4-8cfb-3e7040a18521/LIGHT-MIN-table

2px spacing between table cells